packages/assets/scss/03-organisms/_sticky-toc.scss

```
@@ -563,14 +573,19 @@
   &::before {
     display: block;
     content: '';
-    margin-top: -120px;
-    height: 120px;
+    margin-top: -63px;
```

```suggestion
    margin-top: -63px;
```
why are we changing the spacing? I think this has resulted in lots of visual regressions.

---

This was reverted to the initial setting shown at L.556 and 567, but additional adjustment has been added to maintain the same spacing.